src/components/exchange/ExchangeSearch.js

```diff
@@ -107,6 +121,11 @@ const timingConfig = {
   duration: 300,
 };
 
+const DiscoverButtonShadowsFactory = colors => [
```

Comment:
can we rename this and increase the shadows a lil bit? looking a lil light I think